source/PhysicalQuantity/Force.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace PhpUnitsOfMeasure\PhysicalQuantity;
+
+use PhpUnitsOfMeasure\AbstractPhysicalQuantity;
+use PhpUnitsOfMeasure\UnitOfMeasure;
+use PhpUnitsOfMeasure\HasSIUnitsTrait;
+
+class Force extends AbstractPhysicalQuantity
+{
+	use HasSIUnitsTrait;
+
+	protected static $unitDefinitions;
+
+	protected static function initialize()
+	{
+
+		// newton
+		$newton = UnitOfMeasure::nativeUnitFactory('N');
+		$newton->addAlias('newton');
+		static::addUnit($newton);
+
+		static::addMissingSIPrefixedUnits(
+			$newton,
+			1,
+			'%pN',
+			[
+				'%Pnewton',
+				'%Pnewtons'
+			]
+		);
+
+		// kilonewton
+		$newUnit = UnitOfMeasure::linearUnitFactory('kN', 1000);
+		$newUnit->addAlias('kilonewton');
+		$newUnit->addAlias('kilonewtons');
+		static::addUnit($newUnit);
+
+		// kip
+		$newUnit = UnitOfMeasure::linearUnitFactory('kips', 4.4482216);
```

Comment:
ha, this is a new one on me, I'd not heard of a kip before.

I think what we need here is an `lbf` "pound-force" unit, of which kips is an alias of the "kilopound-force" unit.  That one's tricky though, and if you'd rather I handle it instead, you can just leave this `kips` unit the way it is and I'll figure it out later.